rake/build/package_st2.rake

```diff
@@ -33,6 +33,7 @@ namespace :package do
             opts.components.each do |component|
               execute :cp, "packages/st2/component.makefile ${GITDIR}/#{component}/Makefile"
             end
+	    execute :bash, '$GITDIR/scripts/populate-sha.sh'
```

Comment:
You're mixing both `tabs` with `spaces` in indentation for this line.
Please use spaces only, so it will be consistent with existing ruby code (less confusion in future).